pixart.js

```diff
@@ -1 +1,43 @@
+console.log('script.js loaded...');
+//globals
+var pixelForm = document.getElementById('form');
+var colorBox = document.querySelector('.brush');
+var colorInput = document.getElementById('color-field');
+var colorButton = document.getElementById('set-color');
 
+//create 20 div's with class .square and append them to document.body
```

Comment:
Great job adding comments in your code! This is AWESOME to see. However, just be aware that you need to modify your comments if your code changes as well. In this case we changed 20 divs to 8000 divs through our 6th commit